src/main/java/com/synopsys/integration/alert/web/audit/AuditEntryActions.java

```diff
@@ -113,8 +113,31 @@ public AuditEntryModel get(final Long id) {
         return null;
     }
 
-    public AlertPagedModel<AuditEntryModel> resendNotification(final Long notificationdId, final Long commonConfigId) throws IntegrationException {
-        final NotificationContent notificationContent = notificationManager.findById(notificationdId).orElseThrow(() -> new AlertNotificationPurgedException("No notification with this id exists."));
+    public JobAuditModel getAuditInfoForJob(final Long jobId) {
+        if (jobId != null) {
```

Comment:
What defines jobId? @bamandel may have changed this concept.

Reply:
JobId will be the configurationId